HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/ShareInvoiceBottomView.swift

```diff
@@ -0,0 +1,411 @@
+//
+//  ShareInvoiceBottomView.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import UIKit
+
+class ShareInvoiceBottomView: BottomSheetViewController {
+
+    var viewModel: ShareInvoiceBottomViewModel
+    
+    private lazy var contentStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        return stackView
+    }()
+
+    private lazy var titleView: UIView = {
```

Comment:
[Suggestion]: could you please extract a shared code from `titleView` and `tipView`?